var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -17,14 +17,16 @@ class PyYt(PythonPackage):
     """
 
     homepage = "https://yt-project.org"
-    url = "https://github.com/yt-project/yt/archive/yt-3.5.0.tar.gz"
     git = "https://github.com/yt-project/yt.git"
+    pypi = "yt/yt-4.1.2.tar.gz"
 
-    maintainers = ["qobilidop"]
+    maintainers = ["charmoniumq"]
 
-    version("develop", branch="master")
-    version("develop-4.0", branch="yt-4.0")
+    version("main", branch="main")
+    version("develop", branch="master", deprecated=True)
+    version("develop-4.0", branch="yt-4.0", deprecated=True)
 
+    version("4.1.2", sha256="0ae03288b067721baad14c016f253dc791cd444a1f2dd5d804cf91da622a0c76")
     version("3.6.1", sha256="a1be3ea7e18729d3cd86e9234dc4731bf23200dff3344fa756fe173ea36cc747")
```

Comment:
If you're going to switch to PyPI for all downloads, you'll need to update the checksum for all older versions. Something like:
```console
$ spack checksum py-yt $(spack versions --safe-only py-yt)
```
should do the trick.

Reply:
Still waiting on these new checksums